src/sentry/api/endpoints/system_options.py

```diff
@@ -48,7 +49,7 @@ def get(self, request: Request) -> Response:
 
             # TODO(mattrobenolt): help, placeholder, title, type
             results[k.name] = {
-                "value": options.get(k.name),
+                "value": options.get(k.name) if not self.__is_secret(k) else "[redacted]",
```

Comment:
@rahul-kumar-saini a test case that showcases that certain secrets/tokens are inaccessible from this endpoint would be good to have.